### DISCUSSION TITLE
What is the Clang64 you are going to add build support for?

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#146] [General]
Is it the one provided by llvm.org?

https://releases.llvm.org/download.html

Or is it `clang-cl.exe`?

**Comment by objeck** [comment]
Toolchain already builds under Clang

**Reply by ghost** [reply]
I noticed it on README.md as it's a duplication of the previous `Under the Clang64` and marked as TBH. Now it disappeared.
